client/state/directly/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	initialize as initializeDirectly,
+	askQuestion as askDirectlyQuestion,
+	maximize as maximizeDirectly,
+	minimize as minimizeDirectly,
+	openAskForm as openDirectlyAskForm,
+} from 'lib/directly';
+import {
+	DIRECTLY_ASK_QUESTION,
+	DIRECTLY_INITIALIZE,
+	DIRECTLY_MAXIMIZE,
+	DIRECTLY_MINIMIZE,
+	DIRECTLY_OPEN_ASK_FORM,
+} from 'state/action-types';
+
+export const askQuestion = ( questionText, name, email ) => ( dispatch ) => {
+	dispatch( { type: DIRECTLY_ASK_QUESTION, questionText, name, email } );
+	askDirectlyQuestion( questionText, name, email );
+};
```

Comment:
> I'm not sure where you got this information but unfortunately something got lost in translation

That info came from me. I started out by saying lets do it in middleware then it seemed like it'd be better suited for the action creators because I've seen some hairy situations where actions get dispatched from middleware and cause problems. either way, my bad 😄